packages/labs/src/components/select/select.tsx

```diff
@@ -205,7 +205,7 @@ export class Select<T> extends React.Component<ISelectProps<T>, ISelectState<T>>
                 placement="bottom-start"
                 disabled={disabled}
                 {...popoverProps}
-                className={classNames(listProps.className, popoverProps.className)}
+                className={classNames(className, listProps.className, popoverProps.className)}
```

Comment:
I think it might be better to change https://github.com/invliD/blueprint/blob/c10e8eae814c452399a66cc20c5741810f65ccee/packages/labs/src/components/query-list/queryList.tsx#L155 so that we actually pass through `listProps.className` (right now it's _always_ undefined) -- that would be a smaller change that works for all query lists

Reply:
yes that's where this fix should happen. i just made that change locally for #1950